media-libs/oyranos/oyranos-9999.ebuild

```diff
@@ -80,18 +88,37 @@ multilib_src_configure() {
 	local libdir=$(get_libdir)
 	local mycmakeargs=(
 		-DLIB_SUFFIX=${libdir#lib}
+		-DUSE_SYSTEM_ELEKTRA=YES
+		-DUSE_SYSTEM_YAJL=YES
+		$(cmake-utils_use_find_package cairo)
```

Comment:
If you follow the two other advices, I suggest dropping cmake-utils_use* entirely in favor of less confusing/better copy-paste stuff like:

    -DCMAKE_DISABLE_FIND_PACKAGE_CAIRO=$(usex '!cairo')